src/Parallel/ArrayCollection/Tags/NumberOfElementsTerminated.hpp

```diff
@@ -0,0 +1,16 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/DataBox/Tag.hpp"
+
+namespace Parallel::Tags {
+/// \brief A counter of the number of elements that have marked themselves as
+/// terminated on the node.
```

Comment:
terminated in what sense? the phase or program execution?

Reply:
I guess if the program execution is terminated the value of this tag is ill-defined 🙃 but I'll be explicit